### PR TITLE
Register Subdomain: ckateptb

### PR DESCRIPTION
**Pull request by CKATEPTb**
Added the domain: ckateptb

### CODE REVIEW

File: domains/ckateptb.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "CKATEPTb",
+        "email": "taksis@ex.ua"
+      },
+
+      "record": {
+        "A": ["65.108.238.23"]
```
Comment (WilliamDavidHarrison) [inline domains/ckateptb.json:9]: 404?

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/ckateptb.json (hunk 1 above)
Comment (CKATEPTb) [reply domains/ckateptb.json:9]: > 404?

I want to host a maven repo and a business card on this domain in the future.

Comment (WilliamDavidHarrison) [reply domains/ckateptb.json:9]: @CKATEPTb We cannot give you the domain until there is content.

Comment (CKATEPTb) [reply domains/ckateptb.json:9]: @WilliamDavidHarrison fixed

Comment (WilliamDavidHarrison) [reply domains/ckateptb.json:9]: @CKATEPTb What port?

Comment (andrewstech) [reply domains/ckateptb.json:9]: > @CKATEPTb What port?

8080

**Review by andrewstech — APPROVED** [review]

**Pull request merged**